js_modules/dagit/packages/core/src/instance/backfill/BackfillPage.tsx

```diff
@@ -182,31 +182,33 @@ export const BackfillPage = () => {
             </tr>
           </thead>
           <tbody>
-            {backfill.assetBackfillData?.assetPartitionsStatusCounts.map((asset) => (
-              <tr key={asset.assetKey.path.join('/')}>
-                <td>
-                  <Box flex={{direction: 'row', justifyContent: 'space-between'}}>
-                    <div>
-                      <a href={assetDetailsPathForKey(asset.assetKey)}>
-                        {asset.assetKey.path.join('/')}
-                      </a>
-                    </div>
-                    <div>
-                      <StatusBar
-                        targeted={asset.numPartitionsTargeted}
-                        requested={asset.numPartitionsRequested}
-                        completed={asset.numPartitionsCompleted}
-                        failed={asset.numPartitionsFailed}
-                      />
-                    </div>
-                  </Box>
-                </td>
-                <td>{asset.numPartitionsTargeted}</td>
-                <td>{asset.numPartitionsRequested}</td>
-                <td>{asset.numPartitionsCompleted}</td>
-                <td>{asset.numPartitionsFailed}</td>
-              </tr>
-            ))}
+            {backfill.assetBackfillData?.assetBackfillStatuses.map((asset) => {
```

Comment:
curious if `assetBackfillData` is actually nullable?

Reply:
ah yeah good catch--I think before we were still returning an asset backfill data object even with the backfill wasn't an asset backfill. I've updated the resolver to return None for non-asset backfills.